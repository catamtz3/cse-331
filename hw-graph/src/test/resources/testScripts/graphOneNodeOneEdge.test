# Create a graph
CreateGraph oneNode

#Add node to the graph
AddNode oneNode a

#Add edge to the graph
AddEdge oneNode a a aa

# Get the nodes of the graph
ListNodes oneNode

# Get the edges of the graph
ListChildren oneNode a