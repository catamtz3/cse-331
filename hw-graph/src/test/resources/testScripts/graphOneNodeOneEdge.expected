# Create a graph
created graph oneNode

#Add node to the graph
added node a to oneNode

#Add edge to the graph
added edge aa from a to a in oneNode

# Get the nodes of the graph
oneNode contains: a

# Get the edges of the graph
the children of n1 in graph1 are: a[aa]]>